dtfs-central-api/src/services/utilisation-reports/payment-matching-tolerance.service/index.ts

```diff
@@ -0,0 +1,21 @@
+import { CURRENCY, PaymentMatchingToleranceEntity } from '@ukef/dtfs2-common';
+import { EntityManager } from 'typeorm';
+import { PaymentMatchingTolerances } from '../../../types/payment-matching-tolerances';
+import { NotFoundError } from '../../../errors';
+
+const mapActivePaymentMatchingToleranceEntitiesToPaymentMatchingTolerances = (
+  activeToleranceEntities: PaymentMatchingToleranceEntity[],
+): PaymentMatchingTolerances => {
+  return Object.values(CURRENCY).reduce((partialTolerances, currency) => {
+    const currencyTolerance = activeToleranceEntities.find((tolerance) => tolerance.currency === currency);
+    if (!currencyTolerance) {
+      throw new NotFoundError(`No tolerance found for currency ${currency}`);
```

Comment:
Why do we want to load every tolerance every time we call this method? Based on the implementation, I think we should only ever need one tolerance per currency, so throwing an error for the `EUR` currency when we're looking at `GBP` payments may be confusing

Reply:
That's a good point. I think originally I wrote it as passing in the tolerances and thought we had cases where we might call this many times in one endpoint so wanted to write it like that so we only fetch the tolerances once. I then realised actually that many check case doesn't actually exist so moved it in so I think that is a good refactor suggestion, I'll change it!